ethportal-api/src/types/content_value/constant.rs

```diff
@@ -0,0 +1,3 @@
+/// The length of the Merkle proof for the inclusion of a block header in a particular epoch
```

Comment:
Well, there's only one constant in this file, but usually in our codebase we've used the filename `constants.rs`, so I'd update it to remain consistent throughout the codebase, plus it seems reasonable that we'll add more constants to this file eventually